basic_flutter/lib/data/employee.dart

```diff
@@ -1,19 +1,63 @@
 class Employee {
-  late String firstName;
-  late String lastName;
-  late int id;
-  late String email;
+  final String imagePath;
+  final String name;
+  final String profession;
+  final String country;
+  final String email;
+  final String phone;
+  final String location;
+  final bool isDarkMode;
 
-  Employee(
-      {required this.firstName,
-      required this.email,
-      required this.id,
-      required this.lastName});
+  const Employee({
+    required this.imagePath,
+    required this.name,
+    required this.profession,
+    required this.country,
+    required this.email,
+    required this.phone,
+    required this.location,
+    required this.isDarkMode,
+  });
```

Comment:
I believe it would be convenient if you add firstName and id because then we would be able to get the response from the server on the update